examples/observability/adot-amp-grafana-for-java/main.tf

```diff
@@ -37,7 +49,7 @@ data "aws_availability_zones" "available" {}
 
 locals {
   name   = basename(path.cwd)
-  region = "us-west-2"
+  region = var.aws_region
```

Comment:
`aws_region` variable is not defined on this PR, is that suppose to be part of https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/617 or something else?

Reply:
Yes indeed, this is part of #617, I'll remove here